VIDEO_1_SCRIPT.md

```diff
@@ -0,0 +1,99 @@
+
+E-commerce, Social media, cloud computing, mobility… Without a doubt, we are definitely living in the Web API era.
+
+## Types of APIs
+
+There are 4 types of APIs: 
+
+- **Web services APIs**: these are the application development interfaces that enable the exchange of information between an application and a web service. (A web service is a software that provides access to a particular service via a URL). 
+
+    There are four types of Web services APIs:
+
+    •	SOAP (Simple Object Access Protocol), a standard protocol for exchanging information and data in XML between two objects.
+
+    •	XML-RPC, a remote procedure call protocol that uses XML as the data format and HTTP calls as the communication system.
+
+    • JSON-RPC, the same protocol as our previously explained XML-RPC, but in JSON format. 
+
+    •	REST (Representational State Transfer), a software architecture for hypermedia systems on the World Wide Web; a REST API uses the HTTP protocol.
+
+- **Library-based APIs**: these types of APIs are those that allow an application to import a library from other software, in order to exchange information. Nowadays, many of the libraries that provide access to products and services are designed in JavaScript.
+
+- **Class-based APIs**: this type of application development interface allows the connection with data around classes, as is usual in object-oriented programming with Java. The Java API uses abstract classes for the creation of applications just like any program developed in this language. These classes provide everything needed to perform all kinds of functions within these applications. The Java development interface is organized into packages and each of those packages, in turn, contains a set of related classes.  
+
+- **Function APIs in operating systems**: software programs are continuously interacting with operating systems (OS). In many cases, the way they do so is through APIs. Operating systems such as Windows have APIs that enable such communication between programs and the OS.
+
+## Benefits of APIs
+
+Let's highlight the many benefits of APIs:
+
+- **Adaptation**: needs change over time and APIs help anticipate changes. When working on this technology, data migration can be better supported, while offering a better review of the information. 
+
+- **Automation**: APIs allow IT teams rather than people to do the work. Through APIs, workflows can be updated so that they can be done in less time while being more productive.
+
+- **Application**: enabling API access to app components provides flexibility in the delivery of services and information.  
```

Comment:
Shouldn't then be Flexibility instead of Application?

```suggestion
- **Flexibility**: enabling API access to app components provides flexibility in the delivery of services and information.  
```